braket_features/Allocating_Qubits_on_QPU_Devices.ipynb

```diff
@@ -10,7 +10,7 @@
     "\n",
```

Comment:
<p>Can we change this section to one of the blue notification boxes we were using  the PL notebooks?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>You mean change the text into the blue box? How about the codes?</p>

Reply:
<p>Just this paragraph. This needs to be highlighted a bit more in case users are confused.</p>